### DISCUSSION TITLE
The app can't connect.

### DISCUSSION THREAD

**Discussion by qcks** [immich-app/immich#13760] [Q&A]
### The bug

The app can't connect.

### The OS that Immich Server is running on

mac

### Version of Immich Server

last

### Version of Immich Mobile App

last

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
null
```

### Your .env content

```Shell
null
```

### Reproduction steps

null

**Comment by danieldietzler** [comment]
At least pretend you're willing to help us in helping you.

**Comment by bo0tzz** [comment]
1. You were just told in https://github.com/immich-app/immich/discussions/13756#discussioncomment-11065860 that this is a network problem, not an Immich one, so this shouldn't be an issue
2. You've provided absolutely zero information here

**Reply by qcks** [reply]
This is because the default behavior of the app market is to update apps to the latest version! I can't connect after switching service address to local LAN address? The previous app version was fine
@danieldietzler

**Reply by danieldietzler** [reply]
Instead of asking for help, you decided to come in and complain about a software being in an unstable state that is clearly labeled as such. That is just annoying. Besides, bo0tzz is correct and those versions are 100% compatible. Immich is not at fault here.
